gateway-queue/src/lib.rs

```diff
@@ -155,13 +146,11 @@ impl Queue for LocalQueue {
         Box::pin(async move {
             let (tx, rx) = oneshot::channel();
 
-            if let Err(_source) = self.0.send(tx) {
-                #[cfg(feature = "tracing")]
-                tracing::warn!("skipping, send failed: {:?}", _source);
+            if let Err(source) = self.0.send(tx) {
+                tracing::warn!("skipping, send failed: {:?}", source);
                 return;
             }
 
-            #[cfg(feature = "tracing")]
             tracing::info!("shard {}/{} waiting for allowance", _id, _total);
```

Comment:
And in method signature
```suggestion
            tracing::info!("shard {}/{} waiting for allowance", id, total);